.drone.yml

```diff
@@ -16,8 +13,17 @@ steps:
       - apk add make
       - make test
 
+  - name: build
+    image: plugins/kaniko-ecr
+    pull: always
+    settings:
+      no_push: true
+    when:
+      event: [pull_request]
```

Comment:
This would enable an unprivileged snyk scan if desired. You would have access to both the Docker file and the oci container tar.

Reply:
With the caveat that we would need to expose a snyk token to PR events

Reply:
This is a public repo so if you just want to snyk container test and report the build result to drone you can I believe. You would only need a token for monitoring the build in a snyk project.

Reply:
It appears that all of these require auth

```
root@0106a2fb1af1:/app# snyk test
`snyk` requires an authenticated account. Please run `snyk auth` and try again.
root@0106a2fb1af1:/app# snyk container test
`snyk` requires an authenticated account. Please run `snyk auth` and try again.
```

Reply:
I might have conflated this with just a default snyk scan then.

Reply:
something for a later date perhaps.